packages/components/README.md

```diff
@@ -13,11 +13,12 @@ At the root of your React app, wrap your app in the `<Provider>` component:
 ```tsx
 // _app.tsx (or the root of your app)
 import { Provider } from '@web3-ui/components';
+import { NETWORK } from '@web3-ui/core';
 
 // Passing in a theme is optional
 function MyApp({ Component, pageProps }) {
   return (
-    <Provider theme={yourChakraTheme}>
+    <Provider theme={yourChakraTheme} network={NETWORKS.mainnet}>
```

Comment:
You can't pass in a `network` prop to a `Provider` imported from the `components` package. We should revert changes made to this file 🙂

Reply:
I did but - it seems like that statement isn't true 🤔 

the Provider component has a network prop in components as well as in hooks

![image](https://user-images.githubusercontent.com/1188186/152288080-722ca888-30b2-40ab-a5b2-56d107959b16.png)